frontend/src/components/pages/UserCodeReviewPerformance/PullRequestReview.jsx

```diff
@@ -44,17 +50,20 @@ export default function PullRequestReview({ review }) {
   }
 
   return (
-    <Tooltip
-      title={
-        <React.Fragment>
-          <Typography>{review.state}</Typography>
-          <em>Timestamp:</em> {formatTimeString(review.submittedAt)}
-        </React.Fragment>
-      }
-    >
-      <Avatar variant="rounded" className={getClassName({ review })}>
-        PR {review.state}
-      </Avatar>
-    </Tooltip>
+    <ThemeProvider theme={theme}>
+      <CssBaseline />
+      <Tooltip
+        title={
+          <React.Fragment>
+            <Typography>{review.state}</Typography>
+            <em>Timestamp:</em> {formatTimeString(review.submittedAt)}
+          </React.Fragment>
+        }
+      >
```

Comment:
If you hover over  `PR changes_requested`, the tool-tip displays like this `changes_requested` please format it such that it appears like this: `changes requested`